extensions/vscode/README.md

```diff
@@ -1,11 +1,68 @@
-# Posit Publisher
+# Posit Publisher - Deploy to Posit Connect
 
-[Posit Publisher](https://github.com/posit-dev/publisher) is an extension for
-[Visual Studio Code](https://code.visualstudio.com/) that provides an easy
-way to deploy to [Posit Connect](https://posit.co/products/enterprise/connect/).
+[Posit Publisher](https://github.com/posit-dev/publisher) is a code-first tool
+with a push-button user interface for deploying the things you build in R and
+Python to [Posit Connect](https://posit.co/products/enterprise/connect/).
+
+Supported content includes:
+
+- Python content: APIs, applications, and notebooks
+- Quarto content: `.qmd` and `.Rmd` files. Support for embedded Shiny apps
+- R content: Shiny apps, RMarkdown, Plumber APIs, and Rmd with embedded
+  Shiny apps
 
 ## Installation
 
-Install Posit Publisher from the
+Install Posit Publisher by using the **Install** button above, or using the
+**Extensions** side bar in [Positron](https://github.com/posit-dev/positron) or
+[VS Code](https://code.visualstudio.com/) and searching for `posit.publisher`.
+
+Posit Publisher is available from the
 [Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=Posit.publisher)
 or from the [Open VSX Registry](https://open-vsx.org/extension/posit/publisher).
+
+## Features
+
+### Code-first deployment
+
+Configure your content and then deploy to Posit Connect with a click.
+
+[Configuration files](https://github.com/posit-dev/publisher/blob/main/docs/configuration.md)
+allow setting up how you want your content to look and run on Connect - from
+choosing a title and description to
+[Runtime process settings](https://docs.posit.co/connect/user/content-settings/#content-runtime).
+
+Keep your content stable with language and tool versions set in your
+configurations — all automatically detected based on your local environment.
+
+Everything is saved in the `.posit/` directory for easy re-deployment and
+collaboration.
+
+### Seamless collaboration
+
+Allow Collaborators to easily pick up where you left off by committing the
+`.posit/` directory to your version control system. Your team can deploy using
+the same deployments and configurations.
+
+### Track all deployments for a project
+
+Easily track all deployments for a project with Posit Publisher.
+
+For example, create separate deployments for staging and production.
```

Comment:
Any more details we can add here? This seems a little sparse / only part of the story.

Also do we want "projects" here. In some ways it's a good use of that: if your'e deploying an API and then a front end that uses that, you can track both of those in the publisher in a way you couldn't really in RStudio before. But also if that's what we're calling `projects` that's not the staging/production thing...

Reply:
Yeah I struggled with this one. Separating out the multiple deployments and the project management features shouldn't be conflated like I was doing here.

Reply:
I split the two concepts into two sections and added a bit more detail
- "Track all content for a project"
  - Highlights the API / front end application use case
- "Multiple deployments and configurations"
  - Highlights the staging / production use case
  
Let me know if you think I captured those well

Reply:
Looks good, thanks!